### ISSUE TITLE
Merge acquisition and process into a single entity

### ISSUE THREAD

**Issue by grololo06** [oceanomics/ecotaxa_dev#367]
It's presently possible to have separate entities, but seldom used.
The query:
`SELECT *
FROM (SELECT "projid", "sampleid", "acquisid", COUNT(*) AS "n"
FROM (SELECT DISTINCT *
FROM (SELECT "projid", "sampleid", "acquisid", "processid"
FROM "obj_head") "dbplyr_006") "dbplyr_007"
GROUP BY "projid", "sampleid", "acquisid") "dbplyr_008"
WHERE ("n" > 1)
`
Shows that on production DB, only a few tuples do not fit, which can be corrected manually.
Technical problems that come to me:
- Merge the "free columns" mappings.
- Ensure consistency at import

**Comment by grololo06** [comment]
@jiho please complete using the different types of subsamples as exposed during June session.

**Comment by jiho** [comment]
`subsample`s can be:
- actual, physical subsamples. Ex: net samples are often separated in two fractions (big stuff and small stuff) before scanning.
- artificial bins for continuously sampling instruments. Those can be 
    * depth bins for profiling instruments Ex: 5m bins for UVP
    * time bins for moored or towed instruments, where a sample may be a day or a deployment period and a subsample would be any smaller relevant partition (an hour, a minute, whatever). Ex: moored IFCB, towed ISIIS.

**Comment by picheral** [comment]
Can also be TIME bins for UVP deployed on moorings.

**Comment by grololo06** [comment]
We need to decide how this change will impact users.

**Comment by picheral** [comment]
All Zooprocess users will not update instantaneously their version to conform with the new rule. We thus have to take the decision and document it in the app. pages where necessary. 
Existing data will be merged in the new entity and the name could be the concatenation of both existing names.
I will modify Zooprocess when the job will be done in Ecotaxa.

**Comment by grololo06** [comment]
Problem with production data (surprise!)

On my clone DB:

```
create temp table proc_acq as select distinct processid, acquisid from obj_head;

SELECT 136081

select acquisid, count(1) as cnt 
from proc_acq group by acquisid
having count(1) > 1

43 rows affected.

select processid, count(1) as cnt 
from proc_acq group by processid
having count(1) > 1
order by cnt desc

8797 rows affected.

66900	1197
147568	200
...```
So  e.g. process 66900 is linked to nearly 1,2K acquisistions

**Comment by grololo06** [comment]
OK decision was made that process_id unicity will not be checked at project level so the clarification is not needed anymore.

**Comment by grololo06** [comment]
Solved in 2.5.0
